include/seqan3/argument_parser/detail/format_parse.hpp

```diff
@@ -736,10 +739,12 @@ class format_parse : public format_base
                                     std::to_string(positional_option_calls.size()) +
                                     "). See -h/--help for more information.");
 
-        if (sequence_container<option_type> && !std::is_same_v<option_type, std::string>) // vector/list will be filled with all remaining arguments
+        if constexpr (sequence_container<option_type> && !std::is_same_v<option_type, std::string>) // vector/list will be filled with all remaining arguments
         {
             assert(positional_option_count == positional_option_calls.size()); // checked on set up.
 
+            value.clear();
```

Comment:
I don't quite understand, why we have two different functions to set a value.

I would have expected to only have one point to change this value.

Reply:
It's once for positional options and once for options